packages/analytics/analytics-metric-provider/src/composables/useMetricCardBuilder.ts

```diff
@@ -1,17 +1,20 @@
-import type { MetricCardDef } from '@kong-ui-public/metric-cards'
+import type { MetricCardDef, MetricCardType } from '@kong-ui-public/metric-cards'
 import type { Ref } from 'vue'
 import { computed } from 'vue'
 import type { ChronologicalMappedMetrics } from './useMetricFetcher'
 import { DEFAULT_KEY } from './useMetricFetcher'
 
 export interface BuilderOptions {
+  cardType: MetricCardType,
   title: Ref<string>,
+  description?: string,
   record: Ref<ChronologicalMappedMetrics>,
   hasError: Ref<boolean>,
   lookupKey?: string,
   sumGroupedValues?: string[],
   increaseIsBad?: boolean,
   formatValueFn?: (rawValue: number) => string,
+  trendRange?: Ref<string> | undefined,
```

Comment:
This should just be `trendRange?: Ref<string>` -- the `?` implies the `| undefined`.